src/test/java/pages/MainPage.java

```diff
@@ -19,24 +20,56 @@
     @FindBy(xpath = "//form/descendant::div[@class='auth-form__label-title']")
     public Text loginViaNickNameMessage;
 
-    @FindBy(className = "cart-form__title_condensed-additional")
-    public Text shoppingCartMessage;
+    @FindBy(linkText = "Зарегистрироваться на Onlíner")
+    public Link registrationLink;
+
+    @FindBy(xpath = "//input[@placeholder='Ник или e-mail']")
+    public Input loginInput;
+
+    @FindBy(xpath = "//input[@type='password']")
+    public Input passwordInput;
+
+    @FindBy(xpath = "//div/button[@type='submit']")
```

Comment:
is it possible to simplify this locator?

Reply:
done